boards/nordic/bm_nrf54l15dk/bm_nrf54l15dk_nrf54l15_cpuapp_s145_softdevice_mcuboot.dts

```diff
@@ -99,6 +99,10 @@
 &cpuapp_sram {
 	status = "okay";
 
+	/* trim down to not overlap with retainedmem */
```

Comment:
/* Override setting in bm_nrf54l15dk_nrf54l15_cpuapp_common.dtsi to adjust sram size to not overlap with retainedmem */